idl/github.com/uber/tchannel/meta.thrift

```diff
@@ -22,7 +39,7 @@ struct VersionInfo {
 }
 
 service Meta {
-    HealthStatus health()
+    HealthStatus health(1: HealthRequest hr)
```

Comment:
@prashantv this needs some basic docs. Without knowing that the parameter is new, I would assume that `hr` is always expected to be non-nil.